templates/post.html

```diff
@@ -0,0 +1,13 @@
+{% extends 'base.html'%} {%block content %}
+<div class="container">
+  <h3>{{post.title}}</h3>
+  <p>{{post.content}}</p>
+  <div class="is-grouped">
+    <a class="btn btn-success btn-sm" href="/post/{{post.id}}/edit">Edit</a>
+    <a class="btn btn-danger btn-sm" href="/post/{{post.id}}/delete">Delete</a>
+
+    <a class="btn btn-primary btn-sm" href="/">Cancel</a>
```

Comment:
This button should say something like 'back to homepage' instead of 'cancel', this way it is more clear to the user what will happen if they click.